ArcadiaParties.Token/TokenService/TokenService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+
+namespace ArcadiaParties.Token
+{
+    public class TokenService : ITokenService
```

Comment:
It'd better be renamed to `OnBehalfOfTokenService`.

Reply:
Renamed